src/main/java/org/folio/search/service/ResourceIdsJobService.java

```diff
@@ -28,15 +29,21 @@ public ResourceIdsJob getJobById(String id) {
 
   @Transactional
```

Comment:
I'm thinking about if this annotation is needed there because all db-related work is done in `resourceIdService.streamResourceIdsForJob` which is already transactional.

Reply:
Yes, here we just save first job. Looks like this annotation is optional here